tracing/dbtracer.go

```diff
@@ -27,6 +28,13 @@ type milestoneTableEntry struct {
 	Location string  `json:"location" akita_data:"index"`
 }
 
+// traceIndexEntry 是trace表的索引结构，只包含session信息
+type traceIndexEntry struct {
+	TableName    string  `json:"table_name" akita_data:"unique"`
```

Comment:
If TableName is "1", trace table is "trace_1", milestone table is "milestone_1". If so, I recommend changing this entry to name "tableNameSuffix". Also add a comment before this to describe with examples.